blueprints/maximum-plaid/index.js

```diff
@@ -0,0 +1,24 @@
+/*jshint node:true*/
+module.exports = {
+  description: 'Installation blueprint for maximum plaid',
+  normalizeEntityName: function () {},
+  beforeInstall: function() {
+    return this.addAddonsToProject({
+      packages: [
+        'ember-d3@~0.2.0',
```

Comment:
I'm not sure we need this, I'd like to use D3 v4 as the base, as it's much more consistent in APIs than v3.x